dist/windows/README.md

```diff
@@ -1,6 +1,6 @@
 # radare2-installer
 Windows installer scripts for radare2
 
-[![Build Status](http://ci.rada.re/buildStatus/icon?job=radare2-win-installer)](http://ci.rada.re/job/radare2-win-installer/)
```

Comment:
all those links are broken. ill update the readmes in a separate pr. but at least the pr should be good to merge now